jobs/JGLOBAL_ATMOS_ANALYSIS_CALC

```diff
@@ -28,16 +28,19 @@ export GPREFIX="${GDUMP}.t${gcyc}z."
 export APREFIX="${RUN}.t${cyc}z."
 export GPREFIX_ENS="${GDUMP_ENS}.t${gcyc}z."
 
-RUN=${rCDUMP} YMD=${PDY} HH=${cyc} declare_from_tmpl -rx COM_OBS
+RUN=${rCDUMP} YMD=${PDY} HH=${cyc} declare_from_tmpl -rx \
+    COMIN_OBS:COM_OBS_TMPL
 
-YMD=${PDY} HH=${cyc} declare_from_tmpl -rx COM_ATMOS_ANALYSIS COM_ATMOS_RESTART
+YMD=${PDY} HH=${cyc} declare_from_tmpl -rx 
```

Comment:
```suggestion
YMD=${PDY} HH=${cyc} declare_from_tmpl -rx \
```
This is the fix needed.

Reply:
@aerorahul 
I have changed line 34 as follows:
 34 YMD=${PDY} HH=${cyc} declare_from_tmpl -rx
 35     COMOUT_ATMOS_ANALYSIS:COM_ATMOS_ANALYSIS_TMPL \
 36     COMIN_ATMOS_RESTART:COM_ATMOS_RESTART_TMPL

Reply:
@aerorahul 
My changes are located the branch: mingshichen-noaa:feature/jgdas_enkf_select_obs_com_in_out

Reply:
please push them